add-ons/grafana/README.md

```diff
@@ -0,0 +1,39 @@
+# Grafana Add-on
+An addon that will deploy grafana with hawkular and prometheus datasources.
+It requires to connect to hawkular-metrics, provided by management-infra addon.
```

Comment:
@gordillo-ramon-redhat , does it mean I need to install `management-infra` add-on first with 
```
$ minishift addon install management-infra
$ minishift addon apply management-infra
```

Reply:
Exactly that.